site/_scss/blocks/_card-section.scss

```diff
@@ -0,0 +1,315 @@
+/* stylelint-disable max-nesting-depth*/
```

Comment:
FWIW, my knowledge of CSS best practices are limited, so I don't think I can give a thorough review of this file.

Reply:
Acknowledge.